genisys/main.py

```diff
@@ -83,14 +83,16 @@ def run(subcommand, args, module):
     if subcommand == "validate":
         validate(module)
     elif subcommand == "install":
-        install_config(args.file, args.root)
+        install_config(yamlParser, args.root)
         # setup commands
         for mod in modulesList:
+            setup = []
+            setup.append(mod.setup_commands)
```

Comment:
This still isn't calling the setup_commands function. It needs to be `mod.setup_commands()`

Reply:
If you're testing locally there's an issue with base.py, you can run `git checkout main -- genisys/modules/base.py` to get the fixed file. Then if you run it you get
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/robert/Development/genisys/genisys/main.py", line 150, in <module>
    main()
  File "/home/robert/Development/genisys/genisys/main.py", line 146, in main
    run(args.command, args, modules)
  File "/home/robert/Development/genisys/genisys/main.py", line 93, in run
    subprocess.run(command, check=False)
  File "/usr/lib64/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib64/python3.12/subprocess.py", line 1808, in _execute_child
    args = list(args)
           ^^^^^^^^^^
TypeError: 'method' object is not iterable
```